crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -1317,4 +1318,172 @@ mod tests {
         let reason = UnableToDecryptReason::UnknownMegolmMessageIndex;
         assert!(reason.is_missing_room_key());
     }
+
+    #[test]
+    fn snapshot_test_verification_level() {
+        let level = VerificationLevel::VerificationViolation;
+        assert_json_snapshot! {
+            serde_json::to_value(&level).unwrap(),
+        }
```

Comment:
Wait a minute, what's going on here?

Why are you calling `serde_json::to_value()` manually? You can just do this:

```suggestion
        assert_json_snapshot!(VerificationLevel::VerificationViolation);
        }
```

This applies to all the places wehre you use `assert_json_snapshot()`.

Reply:
🤯
so much better like that